### PR TITLE
added question sentences

### PR DESCRIPTION
**Pull request by syedatif4118**
I have added some new question sentences! Please check out and if there is any issue please let me know

### CODE REVIEW

File: verbcalc/core/translator/symbols.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def __init__(self):
         self._absolutes = ['absolute of', 'absolute value of']
         self._modulo = ['mod', 'modulo']
         self._roots = ['root of']
-        self._questions = ['what is the result of', 'what is', 'calculate']
+        self._questions = ['what is the result of', 'what is', 'calculate', 'when','if', 'result of','addition of','how do you','find','how much','how much is', 'which number is result of', 'subtraction of','what' ]
```
Comment (ErykPiasecki07) [inline verbcalc/core/translator/symbols.py:29]: Great job, however there are some words that we have to be careful using. 'When' and 'if' are logical connectives and have some deeper meaning that has to be handled separately. I would also opt out from 'addition of', and 'subtraction of', as they suggest the operation that will not be further translated to any symbol. The rest however is very good!
```suggestion
        self._questions = ['what is the result of', 'what is', 'calculate', 'result of', 'how do you', 'find',' how much', 'how much is', 'which number is result of', 'what' ]
```

**Review by ErykPiasecki07 — CHANGES_REQUESTED** [review]

**Review by ErykPiasecki07 — APPROVED** [review]

**Comment by syedatif4118** [comment]
OK, thanks a lot for letting me contribute to your project. I hope I was helpful.
Can you please let me know if there are any other issues where I can contribute? 
Which will help me enhance my skills plus help in more contributions.

**Comment by ErykPiasecki07** [comment]
Great to see you coming back!
All the issues that don't have assignee are free to take! 

Have a nice day!

**Comment by ErykPiasecki07** [comment]
By the way, feel free to add yourself to the recently created AUTHORS.md file! I think contributors like You should be appreciated for their work!

**Comment by syedatif4118** [comment]
Yh sure I will check out the issues and let you know!!

**Pull request merged**